python/ray/train/torch/torch_trainer.py

```diff
@@ -93,6 +93,40 @@ def train_loop_per_worker():
     To save a model to use for the ``TorchPredictor``, you must save it under the
     "model" kwarg in ``Checkpoint`` passed to ``session.report()``.
 
+    .. note::
+        If you are wrapping your ``model`` with ``prepare_model``, save ``model`` to
+        session is equivalent to saving ``model.module`` to session.
```

Comment:
This sentence is unclear to me. What is "saving to session"? Is this `session.report`?